src/event/key.rs

```diff
@@ -94,6 +95,31 @@ impl Key {
   }
 }
 
+impl fmt::Display for Key {
+  fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    match *self {
+      Key::Alt(' ') => write!(f, "<Alt+Space>"),
+      Key::Ctrl(' ') => write!(f, "<Ctrl+Space>"),
+      Key::Char(' ') => write!(f, "<Space>"),
+      Key::Alt(c) => write!(f, "<Alt+{}>", c),
+      Key::Ctrl(c) => write!(f, "<Ctrl+{}>", c),
+      Key::Char(c) => write!(f, "{}", c),
+      Key::Left | Key::Right | Key::Up | Key::Down => write!(f, "<{:?} Arrow Key>", self),
+      Key::Enter
+      | Key::Tab
+      | Key::Backspace
+      | Key::Esc
+      | Key::Ins
+      | Key::Delete
+      | Key::Home
+      | Key::End
+      | Key::PageUp
+      | Key::PageDown => write!(f, "<{:?}>", self),
+      _ => write!(f, "{:?}", self),
+    }
+  }
+}
```

Comment:
Very nice 👌

Reply:
One note about this: there aren't any default keybindings currently bound to function keys, so I wasn't sure whether those should be formatted as `F12` or `<F12>`. If you prefer the latter then you can do away with the `Key::Enter...` case above and make the default case `_ => write!(f, "<{:?}>", self),`.

Reply:
Good to know, thank you.